src/lib/snark_worker/snark_worker.ml

```diff
@@ -1,119 +1,23 @@
-module Prod = Prod
-module Intf = Intf
-module Inputs = Prod.Inputs
+(* NOTE: These documentation might be helpful
+   - https://docs.minaprotocol.com/mina-protocol/snark-workers
```

Comment:
You can use odoc conventions for links: https://github.com/ocaml/odoc/blob/master/doc/examples/odoc_examples.ml#L4